### PR TITLE
move role.name to user_workspace_role.role

### PR DESCRIPTION
**Pull request by karenetheridge**
This removes the role table and collapses the types of roles from 5 to 3:

    conch=> select * from role;
     id |        name        |                                                                                  description
    ----+--------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
      1 | Administrator      | Full-access administrator for the workspace
      2 | Read-only          | Read-only access for the workspace
      3 | Integrator         | Integrator has permissions to use Relays and assign Devices to Racks for the workspace
      4 | DC Operations      | DC Operations
      5 | Integrator Manager | Integrator manager has all the same permissions as the Integrator role, but may also invite new users to a workspace, create sub-workspaces, and modify validation parameters.
    (5 rows)

Now roles are named 'ro', 'rw', and 'admin'.
(This may require adjustments in conch-cli and conch-ui.)

Closes LP-47016848.

### CODE REVIEW

**Comment by sungo** [comment]
Marking this as "breaking" because it changes the syntax for inviting users to a workspace, specifically that the string names for roles have changed. I think there will be some required changes in the shell too.

File: lib/Conch/Plugin/AuthHelpers.pm [hunk 1]
```diff
@@ -18,7 +18,7 @@ sub register ($self, $app, $conf) {
 
 =head2 global_auth
 
-	if($c->global_auth("DC Operations")) {}
+	if ($c->global_auth('admin')) { ... }
```
[the remark this answers is not in the document]
Comment (karenetheridge) [reply lib/Conch/Plugin/AuthHelpers.pm]: oops, I did that right in the migration, but missed it here.

[the remark this answers is not in the document]
Comment (karenetheridge) [reply lib/Conch/Plugin/AuthHelpers.pm]: no, I did intend do to that. this is an example. I figured an example using 'admin' would be more commonly useful than 'rw'.

[the remark this answers is not in the document]
Comment (sungo) [reply lib/Conch/Plugin/AuthHelpers.pm]: I hope not. Admins can inflict all sorts of damage on a workspace. Admins should be minimal, read-write plentiful.

[the remark this answers is not in the document]
Comment (karenetheridge) [reply lib/Conch/Plugin/AuthHelpers.pm]: I couldn't find one example in the code where any now-rw permission was being checked for :) (or perhaps :( )

[the remark this answers is not in the document]
Comment (sungo) [reply lib/Conch/Plugin/AuthHelpers.pm]: Coming soon to a PR near you. We need to get cascading permissions working next and then start adding permissions checks.

**Pull request merged**